packages/web/src/common/components/Checkbox.tsx

```diff
@@ -27,7 +32,12 @@ const Checkbox: React.FC<CheckboxProps> = ({
   disabled = false,
 }) => (
   <CheckboxInner onClick={!disabled ? onClick : undefined} disabled={disabled}>
-    {checked && <Icon type="check" size={16} />}
+    {checked &&
+      (disabled ? (
+        <Icon type="check" size={16} color={colors.GRAY[300]} />
+      ) : (
+        <Icon type="check" size={16} />
+      ))}
```

Comment:
요런 방법은 어떨까 ㅇ.ㅇ (그냥 제안)

```suggestion
    {checked && <Icon type="check" size={16} color={disabled ? colors.GRAY[300] : colors.BLACK} />}
```